contacktbook/LoggerClass.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace contacktbook
+{
+    public class LoggerClass
+    {
+        public void writeLog(string exceptionText)

```

Comment:
Public class member name should starts with Capital letter. Here and everywhere in the C# world.